App/Containers/LaunchScreen.js

```diff
@@ -1,17 +1,41 @@
 import React, { PropTypes, Component } from 'react';
 import { ScrollView, Text, View, Button } from 'react-native';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { createStructuredSelector } from 'reselect';
+
+//Views
+import { AdminPanelButton } from '../Components/AdminPanelButton';
+
+//Selectors
+import { selectAdminList } from '../Selectors/LaunchScreenSelector';
+import { selectSavedUserData, selectIsUserAuth } from '../Selectors/UserAuthSelector';
+
+//Actions
+import { AdminListActions } from '../Redux/AdminListRedux';
 
 // Styles
 import styles from './Styles/LaunchScreenStyles';
 import colors from '../Themes/Colors';
 
-export default class LaunchScreen extends Component {
+
+class LaunchScreen extends Component {
   static propTypes = {
     navigation: PropTypes.object.isRequired,
+    fetchAdminList: PropTypes.func.isRequired,
+    userData: PropTypes.object,
+    isAuth: PropTypes.bool,
+    adminList: PropTypes.object,
   };
 
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.isAuth !== this.props.isAuth && nextProps.isAuth) {
```

Comment:
This `&& nextProps.isAuth` makes no sense at the end. If it is not defined first part will throw error anyway